### PR TITLE
Add course

### PR DESCRIPTION
**Pull request by ananogal**

### CODE REVIEW

File: _data/trainings.yml [hunk 1]
```diff
@@ -45,3 +45,9 @@
   author: Franziska Sauerwein
   description: For experienced developers who want to improve their design skills in object-oriented languages.
   page: /services/training/oo-design
+
+- id: crafting_swift_apps
+  name: Crafting Swift Mobile Apps
+  author: Ana Nogal
+  description: For experienced mobile developers that want to give the next step with their apps and not only build beautiful apps, but also reliable apps.
```
Comment (TobyRet) [inline _data/trainings.yml]: 'For experienced mobile developers that want to **give the next step**' does not make sense.

Maybe you mean ...

 'get to the next step' ?

or more simply 

'For experienced mobile developers that want to build reliable and beautiful apps.

File: _data/trainings_en.yml [hunk 2]
```diff
@@ -45,3 +45,9 @@
   author: Franziska Sauerwein
   description: For experienced developers who want to improve their design skills in object-oriented languages.
   page: /services/training/oo-design
+
+- id: crafting_swift_apps
+  name: Crafting Swift Mobile Apps
+  author: Ana Nogal
+  description: For experienced mobile developers that want to give the next step with their apps and not only build beautiful apps, but also reliable ones.
```
Comment (TobyRet) [inline _data/trainings_en.yml]: Same as above

File: services/training/crafting-swift-mobile-apps.html [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+layout: training
+training: Test driving mobile apps
+---
+
+<!-- Course summary -->
+<div class="row text-center margin-bottom-60">
+    <div class="col-md-6 col-md-offset-3">
+        <h1>Crafting Swift Mobile Apps</h1>
+        <h4 class="margin-bottom-20">Created by Ana Nogal</h4>
+        <p class="margin-bottom-40 training-description"><strong>For experienced mobile developers that want to give the next step with their apps and not only build beautiful apps, but also reliable apps.</strong>
```
Comment (TobyRet) [inline services/training/crafting-swift-mobile-apps.html]: Same as above

File: services/training/crafting-swift-mobile-apps.html [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+          Learn how to TDD your app using Outside-in approach.
```
Comment (TobyRet) [inline services/training/crafting-swift-mobile-apps.html]: Learn how to TDD your app using **the** Outside-in approach.

File: services/training/crafting-swift-mobile-apps.html [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+    <div class="col-md-8 col-md-offset-2">
+        <div class="row margin-bottom-20">
+            <h3>Course Duration</h3>
+	    <p>2 days</p>
+        </div>
+
+	<div class="row margin-bottom-20">
+            <h3>Why attend?</h3>
+            <ul>
+                <li>  Learn how SOLID principles and TDD can help you build code that is easy to add new features.</li>
```
Comment (TobyRet) [inline services/training/crafting-swift-mobile-apps.html]: I'm not sure if 'build code' is the correct choice of words.
'Build apps' or 'create code' or 'write code' might be better alternatives.

'... can help you build code that is easy to add new features' does not make sense.

Another alternative: 

Learn how SOLID principles and TDD can take the pain out of adding new features to an existing code base.

File: services/training/crafting-swift-mobile-apps.html [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+                <li>Test drive the Weather app  </li>
+                <li>Add a new feature to the app</li>
+            </ul>
+        </div>
+
+        <div class="row margin-bottom-20">
+            <h3>Audience</h3>
+            <p>iOS developers that:</p>
+            <ul>
+                <li>Can understand Swift</li>
```
Comment (TobyRet) [inline services/training/crafting-swift-mobile-apps.html]: 'Understand Swift' not 'Can understand Swift'

File: services/training/crafting-swift-mobile-apps.html [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+                <li>Can write and execute unit tests using XCTests</li>
+                <li>Have a basic understanding of mock objects</li>
+                <li>Are beginners at TDD.</li>
```
Comment (TobyRet) [inline services/training/crafting-swift-mobile-apps.html]: You have a period here but not in other list items above

File: services/training/crafting-swift-mobile-apps.html [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+                <li>Have a development environment consisting of:
+                    <ul>
+                        <li>Xcode</li>
+                        <li>Simulator of at least an iPhone 5</li>
+                    </ul>
+                </li>
+                <li>Be able to create projects, build source code and run test cases in their development environment
+                </li>
+            </ul>
+            <p>In addition a distributed version control system such as Git is desirable.</p>
```
Comment (TobyRet) [inline services/training/crafting-swift-mobile-apps.html]: In addition,

File: services/training/crafting-swift-mobile-apps.html [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+<!-- Course description -->
+<div class="row">
+    <div class="col-md-8 col-md-offset-2">
+        <div class="row margin-bottom-20">
+            <h3>Course Duration</h3>
+	    <p>2 days</p>
+        </div>
+
+	<div class="row margin-bottom-20">
+            <h3>Why attend?</h3>
```
Comment (TobyRet) [inline services/training/crafting-swift-mobile-apps.html:31]: I am not sure how this section below relates to the overall course objective to create 'beautiful' apps

Comment (ananogal) [reply services/training/crafting-swift-mobile-apps.html:31]: So the idea is that beautiful apps, are already been made by them: they just need to make these apps reliable

**Comment by cyplo** [comment]
@ananogal @TobyRet - are you happy with the state of this PR right now or need a bit more time ? Asking because I'm not sure if to merge this. Thanks !

**Pull request merged**